cg/store/filters/status_case_filters.py

```diff
@@ -173,6 +173,16 @@ def filter_cases_by_name_search(cases: Query, name_search: str, **kwargs) -> Que
     return cases.filter(Family.name.like(f"%{name_search}%"))
 
 
+def filter_cases_by_pipeline_search(cases: Query, pipeline_search: str, **kwargs) -> Query:
+    """Return cases with pipeline matching the search pattern."""
+    return cases.filter(Family.data_analysis.ilike(f"%{pipeline_search}%"))
```

Comment:
I don't think this should be a strict search. We have several pipelines with similar names and I don't see any merit to returning cases from different pipelines here.
```suggestion
def filter_cases_by_pipeline(cases: Query, pipeline: str, **kwargs) -> Query:
    """Return cases with the given pipeline."""
    return cases.filter(Family.data_analysis==pipeline))
```

Reply:
The original filtering in the cases function uses pattern matching:
```python
case_q = case_q.filter(Family.data_analysis.ilike(f"%{data_analysis}%"))
```

I'm guessing it because the pipeline is provided from the user through the CLI. But I agree, lets use strict matching instead (but this changes the behaviour of the function).

Reply:
There are some tests which explicitly check that it is possible to filter cases by partial pipeline names:
```python
    # WHEN getting active cases by partial data_analysis
    cases = base_store.cases(data_analysis=str(data_analysis)[:-1])
```
These break when changing the filtering in the cases function to only include exact matches. I'll stick to the original behaviour for now.